frontend/src/model/calendar.ts

```diff
@@ -6,4 +8,16 @@ export type PlantEvent = {
     message: string
 }
 
-export type Action = 'water' | 'move' | 'fertilize'
\ No newline at end of file
+export type PlantEventResponse = {
+    date: string
+    plant: string
+    action: Action
+    priority: number
```

Comment:
Tutaj planowaliśmy zrobić priorytet zależny od dni opóźnienia i interwału akcji. Pytanie tylko czy to będzie robione na froncie i back ma wysłać te dane, czy raczej na back, i on zwróci po prostu float.

Najprostszym rozwiązaniem jest obliczeniu priorytetu `daysLate / interval`, i wtedy wszystko co większe od 0, ale mniejsze od 1.0 byłoby zółtym, a wszystko powyżej czerwonym

Reply:
Myślę, że lepiej na froncie obliczać jaki kolor powinien się wyświetlać. Lepiej rozdzielić to na osobne pola daysLate i interval, wtedy gdzieś w innym miejscu będzie można (ale nie trzeba) dokładnie wyświetlić liczbę dni. Będzie po prostu więcej możliwości.